feature/steps/create_account.py

```diff
@@ -0,0 +1,18 @@
+from behave import when, then
+from feature.page_objects.base_page import click_element_by_css_selector, send_keys_by_css_selector, wait_until_alert_is_present, get_alert_text, get_uuid4
+
+UUID = get_uuid4()
```

Comment:
why did you create this variable outside of the step that uses it?
I thought we were going to validate that you can use that string to log in. But that is not what we are doing in our then step.

Maybe you can create a test to validate the login with a user just created. (that would be an e2e test). but for now you can move it inside the when step.